proposals/graphql/proposal/src/main/asciidoc/chapters/about.asciidoc

```diff
@@ -37,27 +37,115 @@ Also see <<appendix_b>>
-The main reasons developers might want to use GraphQL are:
+The main reasons for using GraphQL are:
 
-* Improved data consumption for customers (IoS, Android, Web). Allowing for example to be able to retrieve several types of data in a single request or limiting the response data to exactly the specific data requested.
-* Better analysis of the exhaustiveness of data calls (allowing to know the use of each node) and better manage the deletion of deprecated fields.
+* Improved data consumption on the client-side (IoS, Android, Web). Allowing for example to retrieve several types of data in a single request or limiting the response data to exactly the specific data requested,
+* Better analysis of the exhaustiveness of data calls (allowing to know the use of each node) and better manage the deletion of deprecated fields,
 * Advanced developer experience:
 ** The schema defines how the data can be accessed and serves as the contract between the client and the server. Developer teams on both sides can work without further communication,
-** Native schema introspection enabling to discover the API and to refine the queries on the client-side. This advantage is increased with graphical tools such as https://github.com/graphql/graphiql[GraphiQL] enabling smooth and easy API discovery,
-** On the client-side, the query language provides a lot of flexibility and efficiency enabling developers to adapt to the constraints of their technical environments (IoS, Android, Web).
+** Native schema introspection enabling to discover APIs and to refine the queries on the client-side. This advantage is increased with graphical tools such as https://github.com/graphql/graphiql[GraphiQL] and https://apis.guru/graphql-voyager/[GraphQL Voyager] enabling smooth and easy APIs discovery,
+** On the client-side, the query language provides a lot of flexibility and efficiency enabling developers to adapt to the constraints of their technical environments (IoS, Android, Web) while reducing server round-trips.
+
+[[graphql_and_rest]]
+=== GraphQL and REST
+Before going further, it is important to clearly differenciate GraphQL from REST.
+
+REST stands for "REpresentational State Transfer". It is an architecture style for network-based sofwtare specified by Roy Fielding in 2000 in a https://www.ics.uci.edu/~fielding/pubs/dissertation/fielding_dissertation.pdf[dissertation] defining 6 theoretical constrainsts:
+
+. uniform interface
+. stateless
+. client-server
+. cacheable
+. layered system
+. code on demand (optional).
+
+Many people believe that REST is just doing JSON over HTTP, but REST is fundamentally technically agnostic, it is an architecure style. For instance, it doesn't require to use HTTP. However, it recommends using the maximum capacity of the underlying network protocol to apply the 6 basic principles.This is why its use on HTTP leads to optimize the HTTP semantic with a proper use of verbs (POST, GET, PUT, PATCH, DELETE) and response codes (2xx, 4xx, 5xx).
```

Comment:
Typo:  `architecure` should be `architecture` or `architectural`.

Reply:
Done